apps/services/user-notification/src/app/modules/notifications/notificationsWorker.service.ts

```diff
@@ -29,6 +51,185 @@ export class NotificationsWorkerService implements OnApplicationBootstrap {
+      from: {
+        name: 'Ísland.is',
+        address: 'no-reply@island.is',
+      },
+      to: {
+        name: fullName,
+        address: profile.email,
+      },
+      subject: template.notificationTitle,
+      template: {
+        title: 'Testinga',
+        body: [
+          {
+            component: 'Image',
+            context: {
+              src: join(__dirname, `./assets/images/logo.jpg`),
+              alt: 'Ísland.is logo',
+            },
+          },
+          {
+            component: 'Spacer',
+          },
+          {
+            component: 'Heading',
+            context: {
+              copy: fullName
+                ? isEnglish
+                  ? `Hi ${fullName}`
+                  : `Hæ ${fullName}`
+                : formattedTemplate.notificationTitle,
+            },
+          },
+          {
+            component: 'Copy',
+            context: {
+              copy: formattedTemplate.notificationBody,
+            },
+          },
+          {
+            component: 'Spacer',
+          },
+          {
+            component: 'Button',
+            context: {
+              copy: isEnglish ? 'Open mailbox' : 'Opna pósthólf',
+              href:
+                formattedTemplate.clickActionWeb ??
+                'https://www.island.is/minarsidur/postholf',
+            },
+          },
+          {
+            component: 'Spacer',
+          },
+          {
+            component: 'TextWithLink',
+            context: {
+              small: true,
+              preText: isEnglish ? 'In settings on ' : 'Í stillingum á ',
+              linkHref: 'https://www.island.is/minarsidur/min-gogn/stillingar/',
+              linkLabel: 'Ísland.is',
+              postText: isEnglish
+                ? ', you can decide if you want to be notified or not.'
+                : ' getur þú ákveðið hvort hnippt er í þig.',
+            },
+          },
+        ],
+      },
+    }
+  }
+
+  async handleEmailNotification({
+    profile,
+    message,
+    messageId,
+  }: HandleNotification): Promise<void> {
+    const { nationalId } = profile
+
+    const allowEmailNotification = await this.featureFlagService.getValue(
+      Features.isNotificationEmailWorkerEnabled,
+      false,
+      { nationalId } as User,
+    )
+
+    if (!allowEmailNotification) {
+      this.logger.info('Email notification worker is not enabled for user', {
+        messageId,
+      })
+
+      return
+    }
+
+    // TODO: Add emailNotification check when ready
+    if (!profile.email) {
+      this.logger.info('User does not have email notifications enabled', {
+        messageId,
+      })
+
+      return
+    }
+
+    const [template, individual] = await Promise.all([
+      this.notificationsService.getTemplate(message.templateId, profile.locale),
+      this.nationalRegistryService.getName(profile.nationalId),
+    ])
+
+    const fullName = individual?.fulltNafn ?? ''
+    const isEnglish = profile.locale === 'en'
```

Comment:
We have a `Locale` enum in shared to use instead of hardcoding strings.

Reply:
It is not in shared,it is only an union type and an array. Leaving as is because I would have to create and Locales enum in shared and TS would complain about the enums not being the same since profile.locale is of UserProfileLocaleEnum, which is generated btw. If I would import from `/api/domains/user-profile/types/locale.enum` it would still complain since the type is generated.

So leaving as is!